types/src/database_sql.rs

```diff
@@ -1555,6 +1558,74 @@ impl From<BasicWord> for AnnotatedForm {
     }
 }
 
+// Loader to get each chapter id from the database regardless if it's a subchapter or not.
+#[async_trait]
+impl Loader<CollectionChapter> for Database {
```

Comment:
Name your ID type more specifically based on the operation you're performing, ideally with the name of the result type coming first. Instead of `CollectionChapter` as you have, which is just the type of return value, I would go for something like `ChaptersInCollection`.